juju-charms/charm-trilio-data-mover-api/src/files/trilio/trilio-data-mover-api.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Datamover API service
+
+[Service]
+User = root
```

Comment:
This need to be run as nova or dmapi user.

Reply:
Ok, I will add dmapi user then.